games-puzzle/nudoku/nudoku-5.0.0.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools
+
+DESCRIPTION="ncurses based sudoku game"
+HOMEPAGE="https://jubalh.github.io/nudoku/"
+SRC_URI="https://github.com/jubalh/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="cairo"
+
+BDEPEND="virtual/pkgconfig"
+DEPEND="
+	cairo? ( x11-libs/cairo )
+	>=sys-devel/gettext-0.20
```

Comment:
This gets inherited to runtime. Is gettext required in runtime, or is it simply executed during building the package? Let me know and I'll move the dependency.

Reply:
Hmm I *think* only during build.